src/components/Select/selectUtils.tsx

```diff
@@ -4,15 +4,23 @@ import type { SelectOption } from './Select';
 export const buildOptions = (options: SelectOption[]): JSX.Element[] => {
   if (options.length === 0) return [];
 
-  return options.map(({ value, label }: SelectOption) => (
+  const formattedOptions = options.map(({ value, label }: SelectOption) => (
     <option key={value} value={value}>
       {label}
     </option>
   ));
+
+  return [
+    <option key='initial' disabled selected value=''>
```

Comment:
I see the following warning in console.  Removing the `selected` attribute resolves the warning while still rendering the placeholder by default.
```suggestion
    <option key='initial' disabled value=''>
```
<img width="757" alt="Screenshot 2024-03-29 at 11 12 17 AM" src="https://github.com/cfpb/design-system-react/assets/2592907/4f98a4c1-7b1c-4b70-993f-5810dde04972">

Reply:
Heyo! Let's fix this warning and then we can merge this in @shindigira 🎉